src/content/components/UnassignedView/UnassignedView.tsx

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+import { BugList } from "../BugList/BugList";
+import { useBugFetcher, BugQuery } from "../../hooks/useBugFetcher";
+import { Container } from "../ui/Container/Container";
+import { Loader, MiniLoader } from "../Loader/Loader";
+import { columnTransforms as cTrans } from "../BugList/columnTransforms";
+
+interface GetQueryOptions {
+  iteration: string;
+  metas: Array<{
+    id: string;
+    component: string;
+    priority?: string;
```

Comment:
nitpick but wonder if this could be typed more tightly?

Reply:
What do you have in mind? This is really a copy of MetaBug that appears everywhere, it's one of the types I had in mind for making a global declarations file. I exported it [here](https://github.com/mozilla/bugzy/pull/233/files#diff-e48222476c5f1fa0f5e7353b918bf31a422e62787ba7799d93d8739080124079) in my latest PR, though that hasn't landed yet

Reply:
Btw this part is really just copied from IterationView.tsx I think. Most of this file is

Reply:
I'm realizing that I don't actually use this `metas` property in this view, so perhaps I can just remove it?

Reply:
Yeah that's fine. Nice catch